### DISCUSSION TITLE
Device cannot be found.

### DISCUSSION THREAD

**Discussion by consciousgaze** [homebridge-plugins/homebridge-august#36] [Q&A]
Hi,
I used your app in my homebridge but it keeps showing the following error 
`[12/16/2022, 12:19:41 AM] Homebridge v1.6.0 (HAP v0.11.0) (August) is running on port 35405.
FetchError: Status 401 (undefined): undefined
[12/16/2022, 12:19:41 AM] [August] Discover Devices: TypeError: Cannot convert undefined or null to object`

It looks like the device cannot be found. Is there a way to figure out why and fix it?

**Comment by donavanbecker** [comment]
first enable debug mode in the plugins settings to get more logs, then open an issue with info about you setup
